### DISCUSSION TITLE
Co-channel interference help

### DISCUSSION THREAD

**Discussion by R1ch1e-Lee** [rtl-airband/RTLSDR-Airband#448] [Q&A]
I currently run 2 different railroad scanner feeds in 2 different cities. My issue is on occasion I will start getting bleedover from local weather transmitters.  The weather sites are approximately 30 miles away from each site. 
I have adjusted the gain down to help out but I wanted to see about other settings (bandwidth, ffw) that I could adjust to help prevent this from  happening and I could then turn my gain back up to help with range.
Railroad frequencies are 161.370, 161.520, 161.190. There are a few weather sites around the area but none really close to my receivers. I think they are on 161.400 and 161.550
Thanks for any suggestions

**Comment by charlie-foxtrot** [comment]
Can you share your config file?

If your target channels are NFM then I would recommend `fft_size = 1024;` and `bandwidth = 5000;`.  If they use CTCSS then that is an easy win.

Have you observed a waterfall of your spectrum when the interference is happening with something like SDR++?  Based on the strength and location of the interference you may be able to adjust your center frequency to move the interference away from your target signals.

**Reply by R1ch1e-Lee** [reply]
Thanks. I will try those 2 settings.  Yes they are all NFM as they are railroad frequencies. 

No CTCSS unfortunately at least not from the engines and portables.  This is on a remote raspberry pi running dietpi so I don't have a good way to check the waterfall.  At least I can't think of a way to check it remotely. 

Here is my config file:

# global parameters go here
log_scan_activity = true;
stats_filepath = "/var/tmp/rtl_airband_stats.txt";

devices:
({
type = "rtlsdr";
index = 0;    #RTL-SDR Number
gain = 29.7;    #Gain Setting
#sample_rate = 2.4;
correction = -1;
mode = "scan";
channels:
(
{
freqs = ( 161.370, 161.520, 161.145, 160.440 );        #Add Freq's Here
modulation = "nfm";
notch = 250.3;
labels = ( "161.370", "161.520", "161.145", "160.440" ); #Label Update
outputs: (
{
disable = true;
type = "icecast";
server = "127.0.0.1";
port = 8000;   #Port Number Setting
mountpoint = "stream.mp3";
name = "Railroad Traffic - Bruceton Sub";   #Name Of Your Choice
genre = "Railroad";
description = "CSX Bruceton Sub - Camden, TN"; #Description Of Your Choice
username = "source";
password = "";   #Icecast2 Password
send_scan_freq_tags = true;
},
{
type = "icecast";
server = "secret";
port = 80;   #Port Number Setting
mountpoint = "secret";
name = "Railroad Traffic - Bruceton Sub";   #Name Of Your Choice
genre = "Railroad";
description = "CSX Bruceton Sub - Camden, TN"; #Description Of Your Choice
username = "source";
password = "";   #Icecast2 Password
send_scan_freq_tags = true;
}
);
}
);
}
);

**Reply by R1ch1e-Lee** [reply]
In the wiki it mentions center frequency is only for multichannel. Is that still the case?  I am  using Scan mode

**Reply by charlie-foxtrot** [reply]
Yes that is the case. Do you want to use scan mode?  Given all your freqs can be picked up simultaneously, multichannel mode will likely give you a better experience

**Reply by R1ch1e-Lee** [reply]
I will try multichannel mode.  Can I use the existing config and just change the mode or will it require changing the channel configuration.  Looking at the wiki it seems it is quite a bit different in the config.    
Thanks

**Reply by charlie-foxtrot** [reply]
no, its a different configuration.  I would recommend setting up a single mixer with an icecast output, then having each of your channels output to the mixer, optionally with different left-right balance
